StyleCop.Analyzers/StyleCop.Analyzers.Test/ReadabilityRules/SA1130UnitTests.cs

```diff
@@ -1010,5 +983,48 @@ public void Test()
 
             await VerifyCSharpFixAsync(testCode, DiagnosticResult.EmptyDiagnosticResults, fixedCode, CancellationToken.None).ConfigureAwait(false);
         }
+
+        protected virtual DiagnosticResult[] GetExpectedResultCodeFixSpecialCases()
+        {
+            return new[]
+            {
+                Diagnostic().WithLocation(8, 20),
+                Diagnostic().WithLocation(9, 20),
+                Diagnostic().WithLocation(10, 25),
+                Diagnostic().WithLocation(11, 30),
+                Diagnostic().WithLocation(12, 30),
+                Diagnostic(CS1065).WithLocation(12, 53),
+                Diagnostic().WithLocation(13, 30),
+                Diagnostic(CS7014).WithLocation(13, 47),
+                Diagnostic().WithLocation(14, 30),
+                Diagnostic(CS1670).WithLocation(14, 47),
+                Diagnostic().WithLocation(15, 25),
+                Diagnostic(CS1669).WithLocation(15, 42),
+            };
+        }
+
+        protected virtual DiagnosticResult[] GetExpectedResultAfterFixCodeFixSpecialCases()
+        {
+            return new[]
+            {
+                Diagnostic().WithLocation(12, 30),
+                Diagnostic(CS1065).WithLocation(12, 53),
+                Diagnostic().WithLocation(13, 30),
+                Diagnostic(CS7014).WithLocation(13, 47),
+                Diagnostic().WithLocation(14, 30),
+                Diagnostic(CS1670).WithLocation(14, 47),
+                Diagnostic().WithLocation(15, 25),
+                Diagnostic(CS1669).WithLocation(15, 42),
+            };
+        }
+
+        protected virtual DiagnosticResult[] GetExpectedResultVerifyInvalidCodeConstructions()
+        {
+            return new[]
+            {
+                Diagnostic().WithSpan(4, 50, 4, 58),
+                DiagnosticResult.CompilerError("CS1660").WithMessage("Cannot convert anonymous method to type 'EventHandler[]' because it is not a delegate type").WithSpan(4, 50, 4, 62),
```

Comment:
💡 Same here

Reply:
I noticed that the only thing differing was the span, so I simplified to only check the start location. Ok simplification?